src/gtfs/read.rs

```diff
@@ -631,6 +631,26 @@ fn manage_comment_from_stop(
     comment_links
 }
 
+fn manage_comment_from_route(
+    comments: &mut CollectionWithId<objects::Comment>,
+    route: &Route,
+) -> CommentLinksT {
+    let mut comment_links: CommentLinksT = CommentLinksT::default();
+    if let Some(desc) = &route.desc {
+        let comment_id = "route:".to_string() + &route.id;
+        let comment = objects::Comment {
+            id: comment_id.clone(),
+            comment_type: objects::CommentType::Information,
+            label: None,
+            name: desc.to_string(),
+            url: None,
+        };
+        comments.push(comment).unwrap();
```

Comment:
Please try to avoid `.unwrap()` or at least comment about why this will never panic.

Reply:
I used an expect, it doesn’t seem common practice, but it does both the work of documenting and unwrapping. And if the unpossible happens, we’ll have some hint.

Reply:
Yes, `.expect()` or document is good (actually, I should push for the use of `.expect()` now so thanks :smile:)